tests/console/commands/test_install.py

```diff
@@ -195,3 +195,15 @@ def test_dry_run_populates_installer(tester: CommandTester, mocker: MockerFixtur
     tester.execute("--dry-run")
 
     assert tester.command.installer._dry_run is True
+
+
+def test_no_path_is_passed_to_installer(tester: CommandTester, mocker: MockerFixture):
+    """
+    The --no-root options is passed to the installer.
+    """
+
+    mocker.patch.object(tester.command.installer, "run", return_value=1)
+
+    tester.execute("--no-path")
+
+    assert tester.command.installer._skip_path is True
```

Comment:
I don't really like this but it's exactly what all of the other tests do so 🤷🏻‍♂️